style.css

```diff
@@ -769,97 +769,169 @@ nav .nav-links-list li a {
   }
 }
 
+#contact {
+  padding: 130px 142px 154px 135px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-start;
+}
+
+.contact-description {
+  width: 39%;
+}
+
+.contact-description h2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #262626;
+}
+
+.contact-form {
+  width: 59%;
+}
+
+.contact-form form {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  width: 100%;
+}
+
+.contact-form form input {
+  width: 100%;
+  padding: 14px 16px;
+  border: none;
+  border-bottom: 1px solid #dbd8d7;
+  background: transparent;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #979493;
+  margin-bottom: 24px;
+}
+
+.contact-form form textarea {
+  width: 100%;
+  padding: 12px;
+  border: none;
+  border-bottom: 1px solid #dbd8d7;
+  background: transparent;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #3c3a39;
+  margin-bottom: 24px;
+  height: 114px;
+  resize: none;
+}
+
+.contact-form form input:focus,
+.contact-form form textarea:focus {
+  border: none;
+  outline: none;
+}
+
+.contact-form form button {
+  background: #ff6b00;
+  padding: 12px;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  border: none;
+}
+
+.contact-form form button:hover {
+  background: #fff;
+  color: #262626;
+  border: 1px solid #262626;
+}
+
+.footer-hr {
+  background: #dfe1e6;
+  height: 1px;
+  margin-bottom: 25px;
+}
+
+#footer footer ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+#footer footer ul li {
+  margin-right: 21px;
+}
+
+#footer {
+  margin-bottom: 30px;
+}
+
+#footer h2 {
+  display: none;
+}
+
 @media screen and (max-width: 768px) {
```

Comment:
> [ ]  **[OPTIONAL]**
It is best to start responsive design using the mobile-first approach to make it easier. 

#### REQUIRED CHANGES 

- Please, kindly add the hover effect on your cards so that it shows the see project button when and when clicked, take the user to the Github repository of your portfolio.
![hover](https://user-images.githubusercontent.com/59397159/165646227-13846ea4-885a-4deb-a20b-09d37c8c26fe.png)

- Your see project button should take a user to your portfolio repo on github. please update it 

![link](https://user-images.githubusercontent.com/59397159/165646525-ce261d3f-aa0d-49f0-bc98-c46ab4d64136.png)